common/test/acceptance/pages/lms/learner_profile.py

```diff
@@ -165,3 +167,93 @@ def age_limit_message_present(self):
         """
         self.wait_for_ajax()
         return self.q(css='#u-field-message-account_privacy').visible
+
+    @property
+    def profile_has_default_image(self):
+        """
+        Return list of visible fields.
+        """
+        self.wait_for_field('image')
+        default_links = self.q(css='.image-frame').attrs('src')
+        return 'default-profile' in default_links[0] if default_links else False
+
+    def mouse_hover(self, element):
+        """
+        Mouse over on given element.
+        """
+        mouse_hover_action = ActionChains(self.browser).move_to_element(element)
+        mouse_hover_action.perform()
+
+    @property
+    def profile_has_image_with_public_access(self):
```

Comment:
Nit: I find this a confusing name for a property. I would be tempted to say that this is a method and not a property.